src/misc/setup.cpp

```diff
@@ -747,6 +747,17 @@ std::vector<Value> Property::GetDeprecatedValues() const
 	return values;
 }
 
+void Property::SetQueueableValue(std::string&& value)
+{
+	assert(!value.empty());
+	queueable_value = std::move(value);
+}
+
+const std::optional<std::string>& Property::GetQueuedValue() const
+{
+	return queueable_value;
+}
```

Comment:
Nice, I'm planning to mark these "queued" changes in the GUI as well when I get to it.